### PR TITLE
Basic label creation functionality

### PR DESCRIPTION
**Pull request by Hammurobbie**
# Description

- Adds Setting button/modal that gives Program Manager ability to create labels, allowing them to chose color and label name.

- Adds mockup-style labels to project dashboard based on "status" boolean that is currently in backend schema. This will be changed in a later canvas.

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/Settings/CreateLabel.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
```
Comment (lawsonarichard) [inline src/components/Settings/CreateLabel.jsx]: Rewrite to SASS, do not use Styled-Components as it doesn't match the stack

File: src/components/Settings/LabelList.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
```
Comment (lawsonarichard) [inline src/components/Settings/LabelList.jsx]: Rewrite to SASS, do not use Styled-Components as it doesn't match the stack

**Review by lawsonarichard — COMMENTED** [review]
Rewrite to SASS, do not use Styled-Components as it doesn't match the stack

**Comment by Cireimu** [comment]
[As a program manager, I'd like to create a status label](https://trello.com/c/9sxjiQhp/28-as-a-program-manager-id-like-to-create-a-status-label)

**Comment by Cireimu** [comment]
[As a program manager, I'd like to color code the labels](https://trello.com/c/fFWqbMxg/30-as-a-program-manager-id-like-to-color-code-the-labels)

**Comment by Cireimu** [comment]
[As a program manager, I'd like to color code the labels](https://trello.com/c/fFWqbMxg/30-as-a-program-manager-id-like-to-color-code-the-labels)

**Comment by Cireimu** [comment]
[As a Program Manager, I'd like to see a preview of a status label while I'm creating it.](https://trello.com/c/Pj3hTdfe/31-as-a-program-manager-id-like-to-see-a-preview-of-a-status-label-while-im-creating-it)

File: src/components/Settings/CreateLabel.jsx (hunk 1 above)
Comment (Cireimu) [reply src/components/Settings/CreateLabel.jsx]: We changed it from styled-components to SASS

File: src/components/ProjectList/ProjectListRow/index.jsx [hunk 3]
```diff
@@ -11,8 +11,13 @@ const cleanName = str => {
 };
 
 const ProjectRow = ({ project }) => {
+  console.log('project list query', project);
```
Comment (Memitaru) [inline src/components/ProjectList/ProjectListRow/index.jsx]: Remove this console log before merging. You should remove console logs before merging whenever possible.

File: src/components/Settings/Settings.jsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    e => {
+      e.preventDefault();
+      executeCreate(label);
+      toggle();
+    },
+    [executeCreate, label, setLabel]
+  );
+
+
+  console.log('label global state', label);
```
Comment (Memitaru) [inline src/components/Settings/Settings.jsx]: Other console log to remove

File: src/components/Settings/Settings.jsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+  return (
+    <div className={bottomLinks}>
+      <Button size="sm" color="secondary" onClick={toggle}>
+        Settings
+      </Button>
+      <Modal isOpen={modal} toggle={toggle} className={className}>
+        <ModalHeader toggle={toggle}>Settings</ModalHeader>
+        <ModalBody>
+          {/* <LabelDiv> */}
```
Comment (Memitaru) [inline src/components/Settings/Settings.jsx]: Are we saving this commented out code for something?

File: src/components/Settings/LabelList.jsx (hunk 2 above)
Comment (Cireimu) [reply src/components/Settings/LabelList.jsx]: We changed it from styled-components to SASS

File: package.json [hunk 6]
```diff
@@ -19,17 +19,20 @@
     "@okta/okta-signin-widget": "^3.4.2",
     "@urql/exchange-graphcache": "^2.0.0",
     "axios": "^0.19.0",
+    "bootstrap": "^4.4.1",
```
Comment (lawsonarichard) [inline package.json:22]: Why is bootstrap being used? Is it only being used for tests?

Comment (Memitaru) [reply package.json:22]: It was the easiest and fastest way to set up the modals they are currently using. They plan to refactor it but needed something up quickly so they could get people working on the rest of the feature.

Comment (lawsonarichard) [reply package.json:22]: Sounds good!

**Review by lawsonarichard — APPROVED** [review]

**Review by Memitaru — APPROVED** [review]

**Review by Cireimu — APPROVED** [review]

**Pull request merged**